service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/dataproccluster/startup.sh

```diff
@@ -0,0 +1,768 @@
+#!/bin/bash
+#
```

Comment:
Please add a comment here (and in the Vertex AI post-startup script) to indicate that changes here should (probably) be reflected in the other script.